content/rs/security/tls-ssl.md

```diff
@@ -6,68 +6,82 @@ alwaysopen: false
 categories: ["RS"]
 aliases: ["/rs/administering/designing-production/security/tls-configuration", "/rs/administering/designing-production/security/client-connections"]
 ---
-Transport Layer Security (TLS), commonly called “SSL”, ensures the privacy of data sent between applications and their Redis databases. TLS also secures connections between Redis Enterprise Software nodes.
+Transport Layer Security (TLS), a successor to SSL, ensures the privacy of data sent between applications and their Redis databases. TLS also secures connections between Redis Enterprise Software nodes.
 
-## TLS authentication
+You can use TLS authentication for one or more of the following types of communication:
 
-You can enable TLS for one or both of the following scenarios:
+- Communication from clients (applications) to your database
+- Communication from your database to other clusters for replication using Replica Of
+- Communication to and from your database to other clusters for synchronization using Active-Active
 
-1. Client-server traffic between your Redis clients and your Redis databases
-1. Replication and synchronization traffic between the nodes of a Redis Enterprise Software cluster
+## Enable TLS for client connections
 
-When you configure `Replica Of` for a database, synchronization traffic flows between the primary instance of the database and the replica instance of the database. You can configure authentication for Replica Of synchronization traffic only, or for all communications, including Replica Of synchronization traffic and data traffic between the database and the clients.
+You can enable TLS by editing the configuration of an existing database (as shown below) or by selecting **Advanced Options** when you are creating a new database.
 
-To enable and configure TLS authentication:
-
-1. In **databases**, either:
-    - Select **Add** ![Add](/images/rs/icon_add.png#no-click "Add") to create a new database.
-    - Select the database that you want to configure and select **Edit** at bottom of the page.
-1. Select **Show advanced options** and enable the **TLS** option .
-    ![database-tls-config](/images/rs/database-tls-config.png "Database TLS Configuration")
-1. Select the TLS scope:
-    - **Require TLS for Replica Of Communications Only** - Only encrypt synchronization traffic.
-    - **Require TLS for All Communications** - Encrypt synchronization traffic and traffic between a client and a server.
+1. Select your database from your database list and navigate to the **configuration** tab.
+1. Select **Edit** at the bottom of your screen.
+1. Enable **TLS**.
+    - **Enforce client authentication** is selected by default. If you choose to change this option, you will enforce encryption without authentication.
```

Comment:
```suggestion
    - **Enforce client authentication** is selected by default. If you unselect this option, you will still enforce encryption, but TLS client authentication will be disabled.
```

Reply:
I think it's really important to be clear when talking about TLS client authentication. "Authentication" by itself is ambiguous, as it may refer to password authentication.